tests/service/connectors/test_queryconfig.py

```diff
@@ -730,10 +746,18 @@ def test_generate_update_stmt(
         assert prepared_request.path == "/3.0/lists/abc/members/123"
         assert prepared_request.headers == {"Content-Type": "application/json"}
         assert prepared_request.query_params == {}
-        assert (
-            prepared_request.body
-            == '{\n  "merge_fields": {"FNAME": "MASKED", "LNAME": "MASKED"}\n}\n'
+        assert json.loads(prepared_request.body) == {
+            "merge_fields": {"FNAME": "MASKED", "LNAME": "MASKED"}
+        }
+
+        # using privacy_request_id placeholder
+        config = SaaSQueryConfig(data_management, endpoints, {}, None, privacy_request)
```

Comment:
spoke with @galvana  about creating tests that go thru the `SaasConnector` to be a bit more robust here, he'll be creating an issue for that